frontend/src/pages/auth/signin/SignIn.tsx

```diff
@@ -25,36 +39,65 @@ const SignIn: React.FC = () => {
     }
 
     const handleSignIn = async () => {
-        const token = await logInService({email: "user@mail.com", password: "password"});
+        const token = await logInService(authCredentials);
         const decodedToken = decodeToken(token);
         const user = await getUserService(decodedToken.userId, decodedToken.userType);
         const loggedUser = createLoggedUser(token, user);
-        if (loggedUser) {
+        if (isValid && loggedUser) {
```

Comment:
you have to firstly check if isValid and then do all your stuff. 
But you send data to server, decode it, send another request, create new model and only then check if it valid